architecture.tex

```diff
@@ -194,6 +194,8 @@ \subsection{Logical Inference and Assumptions}
 \texttt{y} are both created with \texttt{positive=True}, then
 \verb|(x + y).is_positive| will be \texttt{True}.
 
+%% TODO: axe this? Why mention it if we won't discuss it. Maybe move to
+%% future work. - scopatz
```

Comment:
I mentioned it because people who know about it would be wondering about it. But putting it in the future work section sound like a good idea. I'll add something now.